magellan-sample-advanced/src/main/java/com/wealthfront/magellan/sample/advanced/SampleApplication.java

```diff
@@ -0,0 +1,16 @@
+package com.wealthfront.magellan.sample.advanced;
+
+import android.app.Application;
+import android.content.Context;
+
+public class SampleApplication extends Application {
+
+  public static SampleApplication app(Context context) {
+    return (SampleApplication) context.getApplicationContext();
+  }
+
+  public AppComponent injector() {
+    return DaggerAppComponent.create();
```

Comment:
Do `DaggerAppComponent.builder().appModule(new AppModule(this)).build();` in `onCreate`and store the component as a member, then simply return it here in `injector() `